File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):

    cnt = 0
    for i in range(0,len(t) -len(p) + 1):
        if int(t[i:i+len(p)]) <= int(p):
            # print(int(t[i:i+len(p)]))
            cnt += 1
        
    return cnt

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    
    var count = 0;
    for(var i=0; i<=t.length-p.length; i++) {
        if(t.slice(i,i+p.length) <= p) {
            console.log(t.slice(i,i+p.length));
            count += 1;
        }
    }
    
    return count;
}